### PR TITLE
fix Resume cors

### PR DESCRIPTION
**Pull request by mjrb**
add headers
fix bucket options

### CODE REVIEW

**Comment by mjrb** [comment]
deployed and working on test

File: resume.py [hunk 1]
```diff
@@ -60,11 +48,16 @@ def exists(email):
...
+        return add_cors_headers({
+            "statusCode": 200, "body": {
+                "upload": presign("put_object", event, ctx, user),
+                "download": presign("get_object", event, ctx, user),
+                "exists": exists(event["email"])
+            }
+        })
     except ClientError as e:
-        return {"statusCode": 500, "body": "failed to connect to s3" + str(e)}
+        return add_cors_headers({
```
Comment (hemangandhi) [inline resume.py]: Sorta weird that you have to do this... `ensure_schema` should do this for you...

File: resume.py [hunk 2]
```diff
@@ -18,33 +19,20 @@
 def presign(method, event, ctx, user):
     client = boto3.client("s3", **RESUME)
     http_method = "GET"
+    params = {
```
Comment (hemangandhi) [inline resume.py]: Nice refactor!

File: resume.py (hunk 1 above)
Comment (mjrb) [reply resume.py]: you are correct. im thinking about opening an issue to refactor other places like this. like in authorize. also potentially using it as a decorator instead

**Pull request merged**